README.md

```diff
@@ -43,82 +64,106 @@
 5. Retrieve articles with the information about the author attached to each row (there should be 12 rows in the result and around 10 columns, including the article’s title, text, rating, and date as well as the author’s name and sex):
 
     ```postgresql
-    ... here goes your SQL ...
+    select * from articles
+    left join  authors on articles.author_id = authors.id;
     ```
 
 6. To get the twelve rows, you must have used one of the constructions `INNER JOIN`, `LEFT JOIN`, `RIGHT JOIN`, or `FULL JOIN`. How many rows would the other three have returned? First try to think of the answers and then verify them by running the queries (it’s important you understand the results). Put the numbers below:
 
     ```
-    INNER JOIN: ? rows
-    LEFT JOIN: ? rows
-    RIGHT JOIN: ? rows
-    FULL JOIN: ? rows
+    INNER JOIN: 11 rows
+    LEFT JOIN: 12 rows
+    RIGHT JOIN: 13 rows
+    FULL JOIN: 14 rows
     ```
 
 7. Imagine you’re using pagination to display articles showing five articles per page. Retrieve the content for the first page: create a query that would return the latest five articles, ordered from the latest to the earliest.
 
     ```postgresql
-    ... here goes your SQL ...
+    select * from articles
+    order by date
+    limit 5;
     ```
 
 8. Retrieve the content for the second page: articles 6 through 10 (still assuming chronological order).
 
     ```postgresql
-    ... here goes your SQL ...
+    select *
+    from articles
+    order by date
+    offset 5
+    limit 5;
     ```
     
 9. Retrieve the content for the third page: articles 11 through 15 (never mind there are actually only 12 of them currently in the table).
 
     ```postgresql
-    ... here goes your SQL ...
+    select *
+    from articles
+    order by date
+    offset 5
```

Comment:
you use `offset 5` to show articles from 6 to 10, now you need to show from 11 to 15